massgov/mayflower
# DP-9211 [a11y] Remove text alternative from banner image in guide and binder pages

Any PRs being created needs a changelog.txt file before being merged into dev. See: [Change Log Instructions](https://github.com/massgov/mayflower/blob/dev/docs/change-log-instructions.md)


## Description

A banner image in `@organisms/by-template/illustrated-header.twig` for Binder and Guide pages had text alternative though it's a decorative element.

Remove the markup for the text alternative (`role` and `aria-label`) and variables (data) for it.

```
<div class="ma__illustrated-header__image" role="img" aria-label="Forest of the Berkshires"></div>
```

## Related Issue / Ticket

- [JIRA issue](https://jira.mass.gov/browse/DP-9211)

## Steps to Test


1. Go to Binder and Guide pages and check their banner images don't have any text alternative as decorative images.  Find no `role="img"` and `aria-label`.

```
<section class="ma__illustrated-header">
...
  <style>
      .ma__illustrated-header__image {
        background-image: url('../../assets/images/placeholder/800x450.png');
      }

  </style>
  <div class="ma__illustrated-header__image"></div>
  </section>
```

## Screenshots
Use something like [licecap](http://www.cockos.com/licecap/) to capture gifs to demonstrate behaviors.


## Additional Notes:

Anything else to add?

#### Impacted Areas in Application


* 

#### @TODO


*

#### Today I learned...
